spec/support/jasmine-browser.json

```diff
@@ -1,8 +1,7 @@
 {
   "srcDir": ".",
   "srcFiles": [
-    "public/packs/js/application.bundle.js",
-    "public/packs-test/js/application.bundle.js"
+    "public/packs-test/js/application-bundle-*.js"
```

Comment:
i thought you were looking to get rid of `packs-test`

Reply:
We are keeping it for now 👍